collections-plan.md

```diff
@@ -0,0 +1,102 @@
+# Transactions plan
+A list of requirements to support transactions from one collection to another.
+
+### Requirements
+* Change defaults code to allow any defaults and not to force quantity
+* Update local storage to support items being added and removed from collections
+* Implement `moveItem` method 
+
+### Move command description
+
+`moveItem(object: transaction)`
+
+```
+transaction:
+{
+    from: String, //required
+    to: String, //required
+    key:  String, //required
+    decrement: Integer, //default 1
+    fromState: String, //optional
+    toState: String, //optional
+}
+```
+
+## Example Scenarios
+
+#### Scenario 1
+* Shop has 1 shield
+* Player buys shield
+* Shield is added to inventory
+* Shield shows in shop as “owned”
+
+**start collection states:**
+shop [{key: "shield"}]
+inventory []
+
+**Command**
+`collections.moveItem({key: "shield", from: "shop", to: "player", fromState: "owned"])`
```

Comment:
I guess this is meant to say `to: "inventory"`

I understand that fromState changes the state of the thing the item came from, just found it confusing at first

Reply:
you mean the wording "fromState"? Maybe can change that to something more explicit...